test/DynamoCoreWpfTests/PublishPackageViewModelTests.cs

```diff
@@ -116,5 +118,47 @@ public void NewPackageVersionUpload_DoesNotThrowExceptionWhenDLLIsLoadedSeveralT
             Assert.AreEqual(PackageUploadHandle.State.Error, vm.UploadState);
         }
 
+        [Test]
+        [Category("Failure")]
+        [Category("TechDebt")] //when a package is published - it does not load its customNodes. This may be intentional.
+        public void PublishingACustomNodeSetsPackageInfoCorrectly_()
+        {
+            var cnworkspace = this.GetModel().CustomNodeManager.CreateCustomNode("nodeToBePublished", "somecategory", "publish this node") as CustomNodeWorkspaceModel;
+            var inputNode = new Symbol();
+            inputNode.InputSymbol = "input;";
+            cnworkspace.AddAndRegisterNode(inputNode);
+
+            var tempPath = Path.Combine(TempFolder, "nodeToBePublished.dyf");
+            cnworkspace.Save(tempPath, false, this.GetModel().EngineController);
+
+            Assert.IsNull(GetModel().CustomNodeManager.NodeInfos[cnworkspace.CustomNodeId].PackageInfo);
+            Assert.IsFalse(GetModel().CustomNodeManager.NodeInfos[cnworkspace.CustomNodeId].IsPackageMember);
+
+            //now lets publish this node as a local package.
+            var newPkgVm = new PublishPackageViewModel(ViewModel) { CustomNodeDefinitions = new List<CustomNodeDefinition>(){ cnworkspace.CustomNodeDefinition } };
+            newPkgVm.Name = "PublishingACustomNodeSetsPackageInfoCorrectly";
+            newPkgVm.MajorVersion = "0";
+            newPkgVm.MinorVersion = "0";
+            newPkgVm.BuildVersion = "1";
+            newPkgVm.PublishLocallyCommand.Execute();
+
+            Assert.IsTrue(GetModel().GetPackageManagerExtension().PackageLoader.LocalPackages.Any
+                (x => x.Name == "PublishingACustomNodeSetsPackageInfoCorrectly" && x.Loaded == true && x.LoadedCustomNodes.Count ==1));
+
+
+            Assert.AreEqual(new PackageInfo("PublishingACustomNodeSetsPackageInfoCorrectly", new Version(0,0,1))
+                ,GetModel().CustomNodeManager.NodeInfos[cnworkspace.CustomNodeId].PackageInfo);
+            Assert.IsFalse(GetModel().CustomNodeManager.NodeInfos[cnworkspace.CustomNodeId].IsPackageMember);
```

Comment:
Is this not expected to be true?

Reply:
this is currently false, because the packages does not actually get loaded. - It just gets added to the list of loaded packages... not exactly the same thing. Thats why this one is tech debt, my expectations were not what I found.